code/src/components/Task.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+
+import { tasks } from 'reducers/tasks' 
+
+import moment from 'moment'
+import styled from 'styled-components'
+
+import Logo from './assets/delete.svg'
+import { CustomCheckbox } from './CustomCheckbox'
+
+export const Task = ({task}) => {
+
+  const dispatch = useDispatch()
+
+  const handleCheckboxClick = () => {
+    dispatch(tasks.actions.toggleTaskCompleted(task.id))
+  }
+
+  const handleRemoveButton = () => {
+    dispatch(tasks.actions.removeItem(task.id))
+  }
+
+  return (
+    <TaskContainer checked={task.complete}>
+      <CustomCheckbox 
+        isChecked={task.complete} 
+        onChangeHandler={handleCheckboxClick}
+      />
+      <TextContainer>
+        <TaskName>
+          {task.text}
+        </TaskName>
+        <Date>
+        created at {moment(task.id).format('LT')}
+      </Date>
```

Comment:
This shows only the time and not the date 🤔

Reply:
Yes, on purpose, which is why I used LT, and calling it a "daily planner" ;)

Reply:
Haha sorry, my comments are not always as clear as I think they are 🙈 Maybe call the styled component `Time` instead? 😇